server/portal/libs/agave/operations.py

```diff
@@ -40,7 +40,7 @@ def listing(client, system, path, offset=0, limit=100, *args, **kwargs):
     """
     raw_listing = client.files.listFiles(systemId=system,
                                          path=urllib.parse.quote(path),
-                                         offset=int(offset) + 1,
+                                         offset=int(offset),
```

Comment:
Not sure why this was here in the first place.

Reply:
not 100% sure but in v2 file listings returned the current directory followed by the actual contents of that directory (like 'directory/`, `directory/foo`, `directory/bar.txt`).  something to double-check during v3 testing sessions.

Reply:
Ah that makes sense. That is not the case for v3 I believe, as I added this change due to the offset hiding the first file in the folder listing response